### ISSUE TITLE
nesting seems to confuse offsets

### ISSUE THREAD

**Issue by mschipperheyn** [LoicMahieu/react-styled-flexboxgrid#23]
I have the following nested structure. One a large screen, I'm seeing the offset that is applied to a child column being applied to a parent column. It looks like the smOffset of the child is being applied to the parent where it shouldn't be.

    <Row>
        <Col xs={false} sm={1}>
            <FlexCol>
                <XBorder/>
            </FlexCol>
        </Col>
        {/*Offset visible on large screen*/}
        <Col xsOffset={1} xs={11} sm={10}>
            <Row>
                <Col xs={12} smOffset={3} sm={8}>

**Comment by LoicMahieu** [comment]
I am not sure to well understand. Could you please give a repeatable example ? With webpack bin, jsbin, ... ;)
Thanks

**Comment by johnapost** [comment]
Something related is happening to me. The following code will cause this column to be offset 1 at the md size.
```
<Col mdOffset={4} md={4} smOffset={1} sm={10} xs={12}>
```

Changing it to the following worked:
```
<Col xs={12} smOffset={1} sm={10} mdOffset={4} md={4}>
```

So it seems like the order the props matters. Is this intended?

**Comment by LoicMahieu** [comment]
@johnapost Yep order of the props seems to matter, see https://github.com/LoicMahieu/react-styled-flexboxgrid/issues/44

**Comment by ahennie** [comment]
@LoicMahieu I guess this should be closed then?
